src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -205,12 +231,14 @@ const MetaTraderConfig = (() => {
                                 showCitizenshipMessage();
                                 is_ok = false;
                             }
+                            if (Client.get('residence') === 'gb' && !/age_verification/.test(response_get_account_status.status)) {
```

Comment:
since we're using `Client.get('residence') === 'gb' && !/age_verification/.test(response_get_account_status.status)` multiple times should we add it as a function in this file and call it?

Reply:
Personally, I think it is simple enough that we don't need to do it.